public/index.html

```diff
@@ -56,6 +56,7 @@
                         <label for="phone_number">Phone number</label>
                         <input class="form-control input-lg" type="phone" name="phone_number" id="phone_number">
                     </div>
+                    <input type="hidden" name="campaign" id="campaign" value="some_campaign">
```

Comment:
Could we hard code this in Express instead so the user can't override by viewing source?